src/components/ButtonDropdown.vue

```diff
@@ -10,7 +10,7 @@
       </span>
     </button>
     <div v-show="isOpen" class="button-dropdown-content" :style="dropdownStyle">
-      <slot></slot>
+      <slot v-if="isOpen"></slot>
```

Comment:
Update: the toolbar needed some tweaking and so along with that I updated the dropdown button to use `v-if` instead of `v-show` and fixed aforementioned can of styling worms over here: https://github.com/aclu-people-power/map/pull/71 . If this leads to the same effect of what you're aiming for here, could probably omit the changes to the button dropdown in this PR -- LMK.